src/components/filters/filter-dropdown/filter-dropdown.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="btn-group" dropdown>
+    <button dropdownToggle type="button" class="filter-dropdown btn dropdown-toggle">{{ selected?.title }} 
+        <span class="caret"></span>
+    </button>
+    <ul *dropdownMenu class="dropdown-menu" role="menu">
+        <li class="dropdown-list-item" *ngFor="let filter of filters" role="menuitem">
+            <a class="dropdown-item" (click)="selectFilter(filter)">
```

Comment:
![image](https://user-images.githubusercontent.com/20795331/27033097-b8988e32-4f6f-11e7-98a4-58e3d00854e6.png)

Text text in the dropdown should probably all be aligned the same regardless of whether the checkmark is there or not